flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Speeds.js

```diff
@@ -31,7 +33,7 @@ class A32NX_Speeds {
         /** Using true fhi for comparison */
         const isTo = fhi === SimVar.GetSimVarValue("L:A32NX_TO_CONFIG_FLAPS", "number");
         /** Change fhi to differentiate between 1 and 1 + F */
-        if (fhi === 1 && SimVar.GetSimVarValue("TRAILING EDGE FLAPS LEFT ANGLE", "degrees") < 9.99) {
+        if (fhi === 1 && SimVar.GetSimVarValue("FLAPS HANDLE INDEX", "Enum") === 1) {
```

Comment:
Are you sure this works? This code in the past ensured we are able to differentiate between F1 and F1 + F.

Can we simplify this process?

Reply:
Since #4220 there are now 5 different flap handle index positions, 0 for clean, 1 for 1, 2 for 1+f, 3 for 2 etc.

Reply:
Yep, as @lukecologne already mentioned there are now 5 flap positions. The handle obviously (and therefore local variable) uses the same as before. But the flap handle index of the sim has those 5 positions.

It would be even better to reorder the speeds accordingly and just use the flaps handle index right away. I was just unsure what other calculations might be done with the fhi.

Reply:
iirc the flaps handle index is only used to recognize the config the ac is in.

Reply:
> iirc the flaps handle index is only used to recognize the config the ac is in.

From the `FLAPS HANDLE INDEX` from the sim you can now 1:1 derive in which config you are in. For most situations with is not relevant, but for the upper ECAM and here it's important.